src/current/cockroachcloud/cloud-api.md

```diff
@@ -501,54 +596,71 @@ If the request was successful, the client will receive a list of all clusters wi
 {
```

Comment:
I don't think we should show example output here.

Reply:
Swapped to a high level representation, instead, representing real output from my testing

```
{
  "clusters": [
    {<cluster_object1>},
    {<cluster_object2>},
    {<cluster_object3>}
  ],
  "pagination": null
}     
```